Feature/Add MoorDyn capability

pschuenemann opened the pull request:
Hallo,

I realized that the matlab-toolbox, especially Fast2Matlab and Matlab2Fast, could not properly handle MoorDyn input files. Thus, I did some copy and paste and adjustments within the mentioned functions and added the missing capability.

I thought that it may help other users in future and thus request to include it in the NREL's master repo.
If you have any questions or reservations regarding the proposed changes, please let me know.

Best regards,
Paul Schünemann

pschuenemann committed:
Adds parsing of MoorDyn line types, connection properties and line properties table as a formatted table.

pschuenemann committed:
The output parameter section of the MoorDyn input file is often labeled with 'OUTPUTS' (not 'OutList'). So let's FAST2Matlab know this, that it processes the list of outputs correctly.

pschuenemann committed:
add a comment regarding changes on the processing of output parameter list

pschuenemann committed:
The output parameter section of the MoorDyn input file is often labeled with 'OUTPUTS' (not 'OutList').
So let's Matlab2FAST know this, that it processes the list of outputs in the template file correctly.

pschuenemann committed:
Matlab2FAST: Enabling correct writing of MoorDyn tables (line types table, connection properties table, line properties table)

pschuenemann committed:
Matlab2FAST: Adjusting format specifiers, when writing MoorDyn line types, connection properties and line properties tables to get nice readable tables.

ebranlard commented:
Thanks for doing that. In your first commit it seems like the entire file Fast2Matlab file was rewritten, which makes it harder to see the changes. I'm guessing the line endings were replaced. 
I don't know of an easy way to revert this.. 

-  You could maybe revert the file endings by running  `unix2dos   FAST2Matlab.m`, and then recommit, and push to your branch (it will be directly included in that pull request). I think git might figure it out. 
- If that doesn't work, you could try to squash (`git squash`) all yours commits together (including the dos2unix one), and do another request (or force push to your branch, that should then appear in that request).  
- Or, you can restart everything from scratch, use a folder comparison tool between your version and the master version, insert your changes, submit this into one (or several) commits, and do another pull request (or force push).. 

There are other options, using a `git rebase` but I think the options above are easier. Could you maybe give it a try. 

Sorry for the extra work! Let us know if you can try that.

pschuenemann committed:
Change function name from FAST2Matlab to Fast2Matlab to be the same as in NREL's master repo, so that in git/github it's easier to see my changes to this file regarding MoorDyn capability.

pschuenemann commented:
Hey @ebranlard,

you're right. In my first commit my changes are hard to see. Sorry for this.
I checked it, but to my mind the line endings are the same. However, I saw that I somehow changed the name of the function to FAST2Matlab (to be the same as the file name, as MATLAB likes to have it). Due to this I guess git/github thought, that I have rewritten the whole function.
Thus, I changed the function name back to the original one (Fast2Matlab) and committed this change, but this doesn't help. It looks still like I have rewritten the whole function.

To this end, I followed your last approach and restart everything from scratch, which I will pull request in a minute.

Cheers Paul
